src/app/detail/[task_id]/[task_name]/[user_type]/page.tsx

```diff
@@ -0,0 +1,31 @@
+import { AppMargin, SizeofAppMargin } from '@/hoc/AppMargin'
+import { TaskEditor } from '@/app/detail/ui/TaskEditor'
+import { Box, Stack } from '@mui/material'
+import { Sidebar } from '@/app/detail/ui/Sidebar'
+import { taskDetail } from '@/utils/mockData'
+import { UserType } from '@/types/interfaces'
+
+export default function TaskDetailPage({ params }: { params: { task_id: string; task_name: string; user_type: UserType } }) {
+  console.log(params.user_type)
```

Comment:
You're right but following snake case convention because they are routes and even the folder names should have the same names. Following snake case convention for folders that are part of nested routes.